xfuser/model_executor/pipelines/pipeline_cogvideox.py

```diff
@@ -404,6 +405,24 @@ def _init_sync_pipeline(
             )
         return latents, prompt_embeds, image_rotary_emb
 
+
+    def prepare_extra_step_kwargs(self, generator, eta):
```

Comment:
where did you call this function?

Reply:
Here [extra_step_kwargs = self.prepare_extra_step_kwargs(generator, eta)](https://github.com/xdit-project/xDiT/blob/ec182a32ce587435cc1217f701ab3b2a5d1cd1ff/xfuser/model_executor/pipelines/pipeline_cogvideox.py#L248C8-L248C75). In my case, `print(extra_step_kwargs)` always gives me an empty dict.